### ISSUE TITLE
More control over when to notify

### ISSUE THREAD

**Issue by ghost** [KurozeroPB/vscode-commit-reminder#4]
#### My suggestions

- When file changes is above user specified value notify.
- When diff(Insertion + deletion) is above user specified value notify.

Since this was a simple changes I started to work on this. I would like your opinion and guidance.

I took a note to make this extensions a few days back. Then I looked up if there was already one and found yours.
> "Someone already thought of that before me. Nice!!! Might as well work with him to make it better."

😆

**Comment by Pepijn98** [comment]
Hi, those are some pretty nice suggestions. I wasn't sure if there was anything more to add but this is it!
Can you elaborate a little bit on the `diff(Insertion + deletion)`?
I don't fully understand what you mean by this.

Since you already started on these changes I'm open to pull requests.
Just make sure you follow the eslint rules to keep the code the same and clean.

**Comment by ghost** [comment]
Hi. Thanks for the reply. I am glad you considered the suggestions.

By diff (insertion + deletion) I mean the overall repo changes. It could be considered separately.

```
$ git diff --stat
   package-lock.json |  2 +-
   package.json      | 14 ++++++++++++--
   src/extension.ts  | 17 ++++++++++++-----
   3 files changed, 25 insertions(+), 8 deletions(-)
```
Here overall changes is 33. So user sets a minimum overall changes above which user will be notified.

This is my first contribution. So if you have any step I should follow, things I might miss as a first-timer. Please share.

Do you want me to create a separate feature branch or commit changes to master?
Before I create a pull request I will share the branch so you can comment.
Let me know how to proceed.

**Comment by Pepijn98** [comment]
Okay, that clears things up. I like that idea too.
As for creating a pull request, you should create a separate feature branch.
It's a tiny repo so it doesn't really matter but it's best practice so lets just do that.

**Comment by ghost** [comment]
I have created the pull request.

https://github.com/KurozeroPB/vscode-commit-reminder/pull/5

Open for comment.

**Comment by Pepijn98** [comment]
Published extension version 0.2.0 including your pull request and some other minor changes.
Thank you very much for the contribution!

**Comment by ghost** [comment]
Awesome🥰
